pricegraph/src/orderbook.rs

```diff
@@ -578,11 +586,9 @@ impl Orderbook {
         Some((order, user))
     }
 
-    /// Returns `true` if the specified token pair is valid, and `false`
-    /// otherwise.
     fn is_token_pair_valid(&self, pair: TokenPair) -> bool {
```

Comment:
Now that I think about it, maybe even for price estimates we should ignore `buy == sell` token pairs. My main argument here is that (AFAIK) the solver currently ignores these orders anyway, so you can't really specify an overlapping price for them.